saleor/graphql/account/mutations/base.py

```diff
@@ -14,26 +16,23 @@
 from ...core.mutations import (
     BaseMutation,
     ClearMetaBaseMutation,
+    CreateToken,
     ModelDeleteMutation,
     ModelMutation,
     UpdateMetaBaseMutation,
+    validation_error_to_error_type,
 )
 
 BILLING_ADDRESS_FIELD = "default_billing_address"
 SHIPPING_ADDRESS_FIELD = "default_shipping_address"
 INVALID_TOKEN = "Invalid or expired token."
 
 
-def send_user_password_reset_email(user, site):
-    context = {
-        "email": user.email,
-        "uid": urlsafe_base64_encode(force_bytes(user.pk)),
-        "token": default_token_generator.make_token(user),
-        "site_name": site.name,
-        "domain": site.domain,
-        "protocol": "https" if settings.ENABLE_SSL else "http",
-    }
-    emails.send_password_reset_email.delay(context, user.email, user.pk)
+def send_user_password_reset_email(redirect_url, user):
```

Comment:
This function could be moved to `saleor/account/emails`. If we added the ability to trigger a password reset for customers from the dashboard, it could be reused. I would also add short docstring e.g. "Trigger sending a password reset email for the given user."